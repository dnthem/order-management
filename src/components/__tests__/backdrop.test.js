import Backdrop from "../Backdrop";
import renderer from 'react-test-renderer';


describe("Backdrop", () => {

    
    test("should render Backdrop component", () => {
        const component = renderer.create(
            <Backdrop />,
          );
          let tree = component.toJSON();
          expect(tree).toMatchSnapshot();

          
    });
});
