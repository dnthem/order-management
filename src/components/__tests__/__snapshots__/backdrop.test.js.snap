// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="backdrop"
  style={
    {
      "backgroundColor": "rgba(0,0,0,.5)",
      "height": "100%",
      "left": "0",
      "position": "fixed",
      "top": "0",
      "width": "100%",
      "zIndex": "999",
    }
  }
/>
`;

exports[`Backdrop should render Backdrop component 1`] = `
<div
  className="backdrop"
  style={
    {
      "backgroundColor": "rgba(0,0,0,.5)",
      "height": "100%",
      "left": "0",
      "position": "fixed",
      "top": "0",
      "width": "100%",
      "zIndex": "999",
    }
  }
/>
`;
